templates/EventList.phtml

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="utf-8"/>
+    <title>Admin Login</title>
+    <link rel="stylesheet" type="text/css" href="css/genericStyles.css"/>
+    <link rel="stylesheet" type="text/css" href="css/eventList.css">
+    <script defer src="js/eventList.js"></script>
+</head>
+<body>
+    <main>
+        <h1 id="eventTitle">Events</h1>
+        <div class="event">
+            <div class="top">
+                <div class="upper">
+                    <div class="infoBox">
+                        <span><strong>Name</strong></span>
+                        <span >Boobies</span>
+                    </div>
+                    <div class="infoBox">
+                        <span><strong>Type</strong></span>
+                        <span>Taster Session</span>
+                    </div>
+                    <div class="infoBox">
+                        <span><strong>Date</strong></span>
+                        <span>23/06/2019</span>
+                    </div>
+                </div>
+
```

Comment:
Whitespace.

Reply:
Apparently this was fixed, but Felix never mentioned it